_envcommon/services/ecs-eop-tileserver.hcl

```diff
@@ -0,0 +1,222 @@
+
+  mock_outputs = {
+    listener_arns = {
+      80  = "arn:aws:elasticloadbalancing:us-east-1:123456789012:listener/app/mock-alb/50dc6c495c0c9188/f2f7dc8efc522ab2"
+      443 = "arn:aws:elasticloadbalancing:us-east-1:123456789012:listener/app/mock-alb/50dc6c495c0c9188/f2f7dc8efc522ab2"
+    }
+  }
+  mock_outputs_allowed_terraform_commands = ["validate", ]
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# Locals are named constants that are reusable within the configuration.
+# ---------------------------------------------------------------------------------------------------------------------
+locals {
+  source_base_url = "git::git@github.com:gruntwork-io/terraform-aws-service-catalog.git//modules/services/ecs-service"
+
+  # Automatically load common variables shared across all accounts
+  common_vars = read_terragrunt_config(find_in_parent_folders("common.hcl"))
+
+  # Extract the name prefix for easy access
+  name_prefix = local.common_vars.locals.name_prefix
+
+  # Automatically load account-level variables
+  account_vars = read_terragrunt_config(find_in_parent_folders("account.hcl"))
+
+  # Extract the account_name for easy access
+  account_name = local.account_vars.locals.account_name
+
+  # Automatically load region-level variables
+  region_vars = read_terragrunt_config(find_in_parent_folders("region.hcl"))
+
+  # Extract the region for easy access
+  aws_region = local.region_vars.locals.aws_region
+
+  service_name = "eop-tileserver"
+
+  # Define the container image. This will be used in the child config to combine with the specific image tag for the
+  # environment.
+  container_image = "pramsey/pg_tileserv"
+
+  module_config               = read_terragrunt_config("module_config.hcl")
+  config_secrets_manager_arn  = local.module_config.locals.config_secrets_manager_arn
+  container_image_tag         = local.module_config.locals.container_image_tag
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# MODULE PARAMETERS
+# These are the variables we have to pass in to use the module specified in the terragrunt configuration above.
+# This defines the parameters that are common across all environments.
+# ---------------------------------------------------------------------------------------------------------------------
+inputs = {
+  # -------------------------------------------------------------------------------------------------------------------
+  # Cluster and container configuration
+  # -------------------------------------------------------------------------------------------------------------------
+
+  service_name     = local.service_name
+  ecs_cluster_name = dependency.ecs_fargate_cluster.outputs.name
+  ecs_cluster_arn  = dependency.ecs_fargate_cluster.outputs.arn
+
+  launch_type = "FARGATE"
+  task_cpu    = 1024
+  task_memory = 2048
```

Comment:
@smozely thoughts on good initial values for these?

Reply:
Hmm not sure, I think we can drop the amount on the Java side because it was this rivers thing that made it need a lot.